include/mbedtls/des.h

```diff
@@ -133,14 +139,16 @@ void mbedtls_des3_free( mbedtls_des3_context *ctx );
 void mbedtls_des_key_set_parity( unsigned char key[MBEDTLS_DES_KEY_SIZE] );
 
 /**
- * \brief          Check that key parity on the given key is odd.
+ * \brief          Check that the key parity on the given key is odd.
  *
  *                 DES keys are 56 bits long, but each byte is padded with
  *                 a parity bit to allow verification.
  *
- * \param key      8-byte secret key
+ * \param key      The secret key.
+ *                 Must be a readable buffer of length 8 Bytes.
  *
- * \return         0 is parity was ok, 1 if parity was not correct.
+ * \return         \c 0 if the parity was ok.
```

Comment:
If you also correct the documentation I think this could be improved. First, I don't think "ok" is the correct spelling, as it should probably be spelled with capital letters. Secondly, I think "correct" would sound better, but that is a matter of taste.

Reply:
Yes, I will change it.